ui/apps/platform/src/Containers/Vulnerabilities/VulnerablityReporting/ViewVulnReport/ReportJobs.tsx

```diff
@@ -242,7 +243,7 @@ function ReportJobs({ reportId }: RunHistoryProps) {
                         const reportConfiguration: ReportConfiguration = {
                             id: reportConfigId,
                             name,
-                            description,
+                            description: description || '',
```

Comment:
For understanding, would `description ?? ''` protect if value is `null` or `undefined`